### PR TITLE
Run unit tests in Github workflow

### PR DESCRIPTION
**Pull request by Wwwsylvia**
Signed-off-by: Lixia (Sylvia) Lei [lixia_lei@outlook.com](mailto:lixia_lei@outlook.com)

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -32,3 +32,11 @@ jobs:
...
       - name: Build
         run: make build
+      - name: Run unit tests
+        run: make test
+      - name: Upload coverage report
+        uses: actions/upload-artifact@master
+        with:
+          name: notation-coverage-report-${{ github.sha }}
+          path: .cover/
+        if: always()
```
Comment (shizhMSFT) [inline .github/workflows/build.yml:42]: nit: Should we remove this line?

**Review by shizhMSFT — APPROVED** [review]
LGTM

**Review by shizhMSFT — APPROVED** [review]
LGTM

**Review by gokarnm — APPROVED** [review]
LGTM!

**Pull request merged**